scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -56,6 +67,11 @@
     'roles/logging.logWriter',
 ]
 
+SVC_ACCT_ROLES = [
+    'roles/iam.serviceAccountKeyAdmin',
+]
+
+# Required APIs
 REQUIRED_APIS = [
```

Comment:
bigquery API needs to be added. bigquery-json.googleapis.com.

Reply:
Done.